### ISSUE TITLE
The `no-ignored-tests` rule does not allow simple directory references

### ISSUE THREAD

**Issue by jamestalmage** [avajs/eslint-plugin-ava#114]
In AVA, if your test directory is `foo/test`, you can just add this to the config:

``` js
{
  "ava": {
     "files": ["foo/test"],
  }
}
```

AVA detects that `foo/test` is a directory, and behaves as expected (everything within the directory is a test). To get this plugin to behave, you have to convert it to a glob

**Comment by jamestalmage** [comment]
We should probably export `ava/lib/ava-files` as it's own module and just use that.

**Comment by jamestalmage** [comment]
That said, I'm not sure `ava-files` is going to be easy to export, because: 

- I think there are still lots of improvements to be made to `ava-files`, and separating it out is going to make testing/quick iteration painful. We could try a `mono-repo`, but I know not everyone is a fan of those.
- We don't have great unit tests around `ava-files` yet, it's mostly exercised indirectly by the tests of the modules that use it. (this is a product of the fact that ava-files is basically just code extracted from other internal modules).
- I don't think we want to stabilize the API for `ava-files` yet, so we are subject to some churn in this module if we decide to use it.

**Comment by sindresorhus** [comment]
I know this sounds weird coming from me, but I would just copy-paste the code (and clearly mark it as such) until it's stable enough to export. That way it's stable here until we figure out a good public API for it in AVA.

> We could try a mono-repo, but I know not everyone is a fan of those.

Yes, not a big fan. I'll explain *why* soon: https://github.com/sindresorhus/ama/issues/275

**Comment by sindresorhus** [comment]
This is no longer applicable. The plugin now delegates file classification entirely to AVA's own `eslint-plugin-helper`, so there can be no mismatch between how AVA and the plugin resolve files — they use the same code.
